src/coprocessor/trie/mod.rs

```diff
@@ -125,25 +126,13 @@ impl<F: LurkField> CoCircuit<F> for InsertCoprocessor<F> {}
 /// Add the `Trie`-associated functions to a `Lang` with standard bindings.
 // TODO: define standard patterns for such modularity.
 pub fn install<F: LurkField>(s: &mut Store<F>, lang: &mut Lang<F, TrieCoproc<F>>) {
+    lang.add_binding((user_sym("trie-new"), NewCoprocessor::default().into()), s);
     lang.add_binding(
-        (
-            sym!("lurk", "trie", "new"),
-            NewCoprocessor::default().into(),
-        ),
+        (user_sym("trie-lookup"), LookupCoprocessor::default().into()),
         s,
     );
     lang.add_binding(
-        (
-            sym!("lurk", "trie", "lookup"),
-            LookupCoprocessor::default().into(),
-        ),
-        s,
-    );
-    lang.add_binding(
-        (
-            sym!("lurk", "trie", "insert"),
-            InsertCoprocessor::default().into(),
-        ),
+        (user_sym("trie-insert"), InsertCoprocessor::default().into()),
```

Comment:
I don't think this PR needs to or should change the identity of the symbol created here. It should still be `.lurk.trie.insert` (and same goes for `'new` and `lookup`). We might later decide the whole `trie` package does not belong in the `.lurk` package, in which case that can be changed. Meanwhile, the idea is that this may actually end up serving as a language-level primitive, and that is the premise of this naming.

Reply:
The main point I'm making here is that revisiting that decision and instead changing the substance of this coprocessors work is orthogonal to packages, and this PR should not be asserting such opinions.

Reply:
It's failing to read on the tests because there's no `trie` package

Reply:
This therefore reveals a usage issue that needs to be solved. I *think* you're implicitly suggesting that the coprocessor setup code needs to also define the package. In the current implementation, do packages need to be defined before symbols are interned in them — or just before such symbols are later read? Either way, this is a good chance to initialize the usage pattern we intend.

Reply:
The package needs to be available for reading

Reply:
The previous code constructs symbols from paths. The way I see it, this *should* also require the package to exist. It is just skipping the most superficial syntax-parsing step of reading but should conceptually still be 'reading' in the sense that matters for symbol resolution.

I'm not quite sure whether `sym!` as used in the original code would fail because the package is missing. If not, we should probably modify it so it does. Then we should modify the coprocessor setup code here so that it *doesn't* fail because the package is defined (here).

Reply:
I changed the `install` function to also add a custom `Package` to a mutable `State`